tensorflow_asr/models/transducer/streaming_conformer.py

```diff
@@ -0,0 +1,134 @@
+                 encoder_trainable: bool = True,
+                 prediction_embed_dim: int = 512,
+                 prediction_embed_dropout: int = 0,
+                 prediction_num_rnns: int = 1,
+                 prediction_rnn_units: int = 320,
+                 prediction_rnn_type: str = "lstm",
+                 prediction_rnn_implementation: int = 2,
+                 prediction_layer_norm: bool = True,
+                 prediction_projection_units: int = 0,
+                 prediction_trainable: bool = True,
+                 joint_dim: int = 1024,
+                 joint_activation: str = "tanh",
+                 prejoint_linear: bool = True,
+                 postjoint_linear: bool = False,
+                 joint_mode: str = "add",
+                 joint_trainable: bool = True,
+                 kernel_regularizer=L2,
+                 bias_regularizer=L2,
+                 name: str = "streaming_conformer",
+                 **kwargs):
+
+        self.streaming = True # Hardcoded value. Initializes Conformer with `streaming = True`.
+        super(StreamingConformer, self).__init__(
+            vocabulary_size=vocabulary_size,
+            encoder_subsampling=encoder_subsampling,
+            encoder_positional_encoding=encoder_positional_encoding,
+            encoder_dmodel=encoder_dmodel,
+            encoder_num_blocks=encoder_num_blocks,
+            encoder_head_size=encoder_head_size,
+            encoder_num_heads=encoder_num_heads,
+            encoder_mha_type=encoder_mha_type,
+            encoder_depth_multiplier=encoder_depth_multiplier,
+            encoder_kernel_size=encoder_kernel_size,
+            encoder_fc_factor=encoder_fc_factor,
+            encoder_dropout=encoder_dropout,
+            encoder_trainable=encoder_trainable,
+            prediction_embed_dim=prediction_embed_dim,
+            prediction_embed_dropout=prediction_embed_dropout,
+            prediction_num_rnns=prediction_num_rnns,
+            prediction_rnn_units=prediction_rnn_units,
+            prediction_rnn_type=prediction_rnn_type,
+            prediction_rnn_implementation=prediction_rnn_implementation,
+            prediction_layer_norm=prediction_layer_norm,
+            prediction_projection_units=prediction_projection_units,
+            prediction_trainable=prediction_trainable,
+            joint_dim=joint_dim,
+            joint_activation=joint_activation,
+            prejoint_linear=prejoint_linear,
+            postjoint_linear=postjoint_linear,
+            joint_mode=joint_mode,
+            joint_trainable=joint_trainable,
+            kernel_regularizer=kernel_regularizer,
+            bias_regularizer=bias_regularizer,
+            streaming=self.streaming,
+            name=name,
+            **kwargs
+        )
+        self.dmodel = encoder_dmodel
+        self.time_reduction_factor = self.encoder.conv_subsampling.time_reduction_factor
+
+    def _build(self, input_shape, prediction_shape=[None], batch_size=None):
```

Comment:
@andreselizondo-adestech Please rename this to `def make(...)` 😄

Reply:
Done as well :)